crates/forge/tests/integration/meta_tx_v0.rs

```diff
@@ -1,12 +1,30 @@
 use forge_runner::forge_config::ForgeTrackedResource;
 use indoc::indoc;
+use scarb_api::ScarbCommand;
+use semver::Version;
 use std::path::Path;
 use test_utils::runner::{Contract, assert_passed};
 use test_utils::running_tests::run_test_case;
 use test_utils::test_case;
 
+// TODO(#3704) Remove scarb version check
+fn skip_if_scarb_lt_2_11_0() -> Option<String> {
+    match ScarbCommand::version().run() {
+        Ok(version_info) if version_info.scarb < Version::new(2, 11, 0) => {
+            Some("[IGNORED] `meta_tx_v0` syscall is not supported in Scarb < 2.11.0".to_string())
+        }
+        _ => None,
```

Comment:
I'd just panic in case getting scarb version failed i.e. unwrap the result. If we are unable to get version from scarb we are in some kind of incorrect tests setup/state anyway.